test/molecule-role/molecule/default/prepare.yml

```diff
@@ -133,12 +148,35 @@
   vars_files:
     - common_vars.yml
   tasks:
+    - name: Add testagent.com to /etc/hosts
+      become: true
+      lineinfile:
+        path: /etc/hosts
+        line: "{{ hostvars['receiver']['ansible_host'] }} test-stackstate-agent.sts"
     - name: Install stackstate-agent
       shell: "curl -o- {{agent_repo_url}}/install.sh | bash"
       environment:
         STS_API_KEY: "API_KEY"
         STS_URL: "http://{{ hostvars['receiver']['ansible_host'] }}:7077/stsAgent"
         CODE_NAME: "{{agent_current_branch}}"
+    - name: Replace STS URL to https URL
+      become: true
+      replace:
+        path: /etc/stackstate-agent/stackstate.yaml
+        regexp: 'sts_url: http://.*:7077/stsAgent'
+        replace: 'sts_url: https://test-stackstate-agent.sts/stsAgent'
+    - name: Skip SSL Validation
```

Comment:
Should we add the parameter `SKIP_SSL_VALIDATION` to the installation script ?
So we don't need this ansible task and the next one

Reply:
yeah true. added in the install script.